src/routes/(admin)/entity/[action]/[id]/[entityAction]/[entityId]/EntityRecordForm.svelte

```
@@ -0,0 +1,139 @@
+	const {
+		data
+	}: {
+		data: {
+			form: SuperValidated<Infer<EntityRecordFormSchema>>;
+			entityAction: 'add' | 'edit';
+			entityTypeId: string;
+			entity: Partial<Infer<EntityRecordFormSchema>> | null;
+			entityType: { id: number; name: string } | null;
+			currentUser: any;
+		};
+	} = $props();
```

ESLint

[error] 'currentUser' in 'data' is an unused property.

(svelte/no-unused-props)